config/webpack.common.js

```diff
@@ -128,9 +128,9 @@ module.exports = {
       {
         test: /\.ts$/,
         loaders: [
+          '@angularclass/hmr-loader',
           'awesome-typescript-loader',
-          'angular2-template-loader',
-          '@angularclass/hmr-loader'
+          'angular2-template-loader'
```

Comment:
thanks, I'll update angular2-hmr-loader to also emit ts code later if devs want it to transform ts code rather than es5